dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Action/AIEntity.cs

```diff
@@ -112,10 +112,10 @@ public class ClientCitationAppearance
         public string Abstract { get; set; } = string.Empty;
 
         /// <summary>
-        /// The encoding format used for the icon.
+        /// Optional. Encoding format of the `citation.appearance.text` field.
```

Comment:
Update comment for 'Text' property above to reflect that it won't be ignored in Teams as it depends on the value set here